src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -409,13 +417,90 @@ extension ExposureSubmissionCoordinator {
 		navigationController?.present(alert, animated: true, completion: nil)
 	}
 
+	
 	func showThankYouScreen() {
-		let vc = createSuccessViewController()
-		push(vc)
-	}
+		let thankYouVC = ExposureSubmissionThankYouViewController { [weak self] in
+			self?.showSymptomsScreen()
+		} onSecondaryButtonTap: { [weak self] in
+			self?.presentThankYouCancelAlert()
+		} presentCancelAlert: { [weak self] in
+			self?.presentThankYouCancelAlert()
+		}
 
-	// MARK: - Private
+		push(thankYouVC)
 
+	}
+	
+	// MARK: - Internal
+	
+	func presentPositiveTestResultCancelAlert() {
```

Comment:
is internal here correct?
```suggestion
	private func presentPositiveTestResultCancelAlert()
```

Reply:
Let me check...

Reply:
Refactored to private 👍